rdb/src/main/java/org/openmbee/sdvc/rdb/repositories/commit/CommitDAOImpl.java

```diff
@@ -90,7 +90,7 @@ public PreparedStatement createPreparedStatement(Connection connection)
             currentExtraCol++;
         }
         if (timestamp != null) {
-            query.append(" AND date_trunc('milliseconds', timestamp) <= ?");
+            query.append(" AND timestamp <= ?");
```

Comment:
The date_trunc function was added to the query to resolve an issue with time comparisons in postgres. IIRC, it had to do with timestamp being stored in microseconds. Maybe we need to add a conditional here...

Reply:
i think the only time this would cause a problem is when we take a timestring from the user (the only place is a maxTimestamp query on commits, and it'll have to be exact), otherwise the tests for getting branch history and element at commit works in both. there isn't an equivalent function in mysql, so maybe we can just make a note of it and try to add it back if problems occur

Reply:
cleaned up timestamps